macros/SectionOnPage.ejs

```diff
@@ -5,11 +5,30 @@
 //
 // $0   Page path
 // $1   Section name
+// $2   Element name to wrap the page title (not the section name) in;
+//      leave blank or don't specify this parameter to not wrap the title.
+//      Don't include the angle brackets! Only specific tags are permitted;
+//      they're in the allowedWrappers array.
+
+var allowedWrappers = [
+    "code",
+    "kbd",
+];
 
 var lang = env.locale;
 var text = "";
 var page = wiki.getPage($0);
-var title = page.title;
+var title = kuma.htmlEscape(page.title);
```

Comment:
If a page doesn't exist, this will output "_SectionName_ in undefined". Not sure if we want to do something about it.